examples/NativeExamples/user_trace_009_from_file.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// This example shows how to use a user_trace and a kernel_trace in the same program.
+
+#include <iostream>
+#include <thread>
+#include <condition_variable>
+#include "..\..\krabs\krabs.hpp"
+#include "examples.h"
+
+static void setup_ps_provider(krabs::provider<>& provider);
```

Comment:
are either of these being used?

Reply:
Whoops.  Nope, leftover garbage.  Originally user kernel_and_user_trace_001 as the base for the file.